packages/nextjs/utils/scaffold-eth/parseAddressTo0x.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @dev Converts an address string to the form of `0x${string}`
+ * @param {string} address - address to be parsed
+ * @return string of the form `0x${string}`
+ */
+export default function parseAddressTo0x(address: string | undefined): `0x${string}` | undefined {
+  if (!address) return undefined;
+  if (address.startsWith("0x")) {
+    return `0x${address.substring(2)}`;
```

Comment:
return address

Reply:
I tried doing but since the return type is `0x{string}` we can't return just address because the param `address` is of type `string`

Reply:
return address as 0x${string}  :)